web-client/src/presenter/actions/TrialSessionWorkingCopy/getTrialSessionWorkingCopyCaseNotesFlagAction.js

```diff
@@ -0,0 +1,13 @@
+import { state } from 'cerebral';
+
+/**
+ * getTrialSessionWorkingCopyCaseNotesFlagAction
+ *
+ * @param {object} providers.get the cerebral get function
+ * @returns {Object} caseNotesFlag
+ */
+
+export const getTrialSessionWorkingCopyCaseNotesFlagAction = ({ get }) => {
```

Comment:
🔧 Do we need an action for this? Can the action that needs this value pull it directly off of state?